setup.sh

```diff
@@ -1,4 +1,16 @@
 #!/bin/bash
+# Install required npm modules
 npm install
-cp examples/.env .env
 
+FILE=.env
+EXAMPLE_FILE=examples/.env
+
+# Create .env file
+touch "$FILE"
```

Comment:
`touch` is useless here as `>>` creates the file if it doesn't exist, but if the file already has some content, it's probably preferable to wipe it (using `: > $FILE`), or we could copy the example file to `.env` if it doesn't exist, and make a more complex utility that searches for a specific line in `$FILE` and replaces it if the user input isn't empty.